.pre-commit-config.yaml

```diff
@@ -0,0 +1,59 @@
+# See https://pre-commit.com for more information
```

Comment:
Instead of pre commit checks we are already using lefthook to do our checks before pushing https://github.com/evilmartians/lefthook
See https://github.com/nginx/agent/blob/v3/lefthook.yml

I think its better to stick with one tool for any precommit/prepush checks

Reply:
no problem, I will remove pre-commit and stick to using lefthook